custom/lib_portal/models/product_template.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _name = "product.template"
+    _inherit = 'product.template'
+    
+    authorName = fields.Text('Author')
```

Comment:
* we had agreed that the author was going to be a many2one field to the res.partner, right?

Reply:
Modified the code.Will commit it